HousingSearchApi/V1/Boundary/Responses/Transactions/Person.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+
+namespace HousingSearchApi.V1.Boundary.Responses.Transactions
+{
+    public class Person
+    {
+        public Guid Id { get; }
+
+        public string FullName { get; }
+
+        private Person(Guid id, string fullName)
+        {
+            Id = id;
```

Comment:
Based on what Phillip was explaining, will we have a person ID for each transaction made? I think it might be best if we rename this to "Sender".